### PR TITLE
Updating small deps

### PR DESCRIPTION
**Pull request by chrisnojima**
@MarcoPolo please take a look at the flow issues on this

### CODE REVIEW

File: shared/.flowconfig [hunk 1]
```diff
@@ -50,4 +49,4 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue
 
 [version]
-0.24.1
+0.25
```
Comment (MarcoPolo) [inline shared/.flowconfig]: we should set this to 0.25.0 just so we don't get another case where npm updates to 0.25.1 and we get a flow error

**Comment by MarcoPolo** [comment]
🎆 🚢 🇮🇹

**Comment by chrisnojima** [comment]
@MarcoPolo yeah we have to explicitly set the record keys, see the update

**Comment by MarcoPolo** [comment]
👍

**Pull request closed without merging**